cmd/kubelet/app/options/options.go

```diff
@@ -71,6 +71,11 @@ type KubeletFlags struct {
 	// If set, kubelet will use this IP address for the node.
 	NodeIP string
 
+	// Whether to run sanity checks on the NodeIP
+	// May be undesired if e.G. the Kubelets address is not assigned to the server
```

Comment:
typo: "e.g."

typo: "kubelet's"

clarify "address is not assigned to an interface, e.g. because it is behind a NAT."